data/human/deep missions.txt

```diff
@@ -2215,7 +2215,7 @@ mission "Deep: Scientist Rescue 1: Reinforcements"
 	
 	on offer
 		conversation
-			`While refueling at the Farpoint naval base, you are approached by two Navy officers who recognize you as Captain <last>. They tell you that Lieutenant Paris contacted them a few days ago, indicating that the rescue fleet was en route and would be reaching the Alnitak system shortly. They inform you that they have requisitioned two Cruisers to assist in the Star Queen's retrieval, and depart for their ships.`
+			`While refueling at the Farpoint Navy base, you are approached by two Navy officers who recognize you as Captain <last>. They tell you that Lieutenant Paris contacted them a few days ago, indicating that the rescue fleet was en route and would be reaching the Alnitak system shortly. They inform you that they have requisitioned two Cruisers to assist in the Star Queen's retrieval, and depart for their ships.`
```

Comment:
As far as I can tell, 'Naval' instead of 'Navy' is inconsistent usage and only in Amazinite's missions does it seem to pop up. I like the sound of 'Navy' better.

Reply:
Based on my knowledge, Naval Stations or Naval Bases are more comonly used (and likely correct.) One such example is Changi Naval Base, Singapore.

Reply:
That it is grammatically correct and commonly used IRL doesn't have a bearing on if it is commonly used in-game, which, from my knowledge, it isn't. I personally have no issue calling a naval base a naval base, though, and using such a term has to start somewhere, so it may as well be here, seeing as i don't agree that "navy" sounds any better in this instance.

Reply:
It's not about whether it's grammatically correct (which it is). It's the fact that every time "Naval" could have been used in this game, apart from one other time it has been 'Navy'. And it's a base of _the Navy_, not any old navy.